packages/devices/origin-device-registry-irec-local-api/migrations/1631576581265-CapacityString.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CapacityString1631576581265 implements MigrationInterface {
+    name = 'CapacityString1631576581265';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `ALTER TABLE "irec_device_registry_device" ALTER COLUMN "capacity" TYPE character varying`
+        );
+    }
```

Comment:
Please create a migration for existing device capacities, it should be simple.

Reply:
Altering types do not delete data, it changes int to string preserving all the data we had before